sns_ik_lib/src/sns_position_ik.cpp

```diff
@@ -21,9 +21,9 @@
 
 #include <iostream>
 
-using namespace sns_ik;
+namespace sns_ik{
```

Comment:
Yeah, but not a very strong one. I believe this method is slightly preferred to be more explicit about showing where the namespace applies for a library.